src/components/AppRoutes.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import {
+  Routes,
+  Route,
+} from 'react-router-dom';
+
+import HomePage from '../pages/HomePage';
+import Reservations from '../pages/Reservations';
```

Comment:
The page you're trying to import is called `MyReservations`, to prevent confusion with the component `Reservation`, please make the change in order for that route to work! :grin:

Reply:
Changes Reservations to MyReservation